packages/flutter/lib/src/material/button.dart

```diff
@@ -87,7 +87,7 @@ class RawMaterialButton extends StatefulWidget {
   /// Defines the default text style, with [Material.textStyle], for the
   /// button's [child].
   ///
-  /// If [textStyle.color] is a [MaterialStateColor], [MaterialStateColor.resolveColor]
+  /// If [textStyle.color] is a [MaterialStateProperty], [MaterialStateProperty.resolve]
```

Comment:
I feel like all of these comments can remain `MaterialStateColor`. It might make things ambiguous by changing it.

Reply:
I want to shift the focus towards the `MaterialStateProperty` interface. `MaterialStateColor` is just a convenient way to implement `MaterialStateProperty<Color>`.